package.json

```
@@ -8,28 +8,28 @@
   },
   "keywords": [
     "interledger",
-    "five-bells",
+    "connector",
     "payments"
   ],
   "author": "Interledger Team <info@interledger.org>",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/interledgerjs/ilp-connector/issues"
   },
-  "main": "src/index.js",
-  "types": "src/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "bin": {
-    "ilp-connector": "./src/index.js"
+    "ilp-connector": "./dist/index.js"
   },
   "scripts": {
-    "start": "node src/index.js",
-    "start-prof": "node --prof --logfile=${CONNECTOR_V8_LOGFILE:-v8.log} src/index.js",
-    "start:watch": "nodemon src/index.js",
-    "build": "npm run schema-to-tsd && npm run compile-ts",
+    "start": "node dist/index.js",
+    "start-prof": "node --prof --logfile=${CONNECTOR_V8_LOGFILE:-v8.log} dist/index.js",
+    "start:watch": "nodemon dist/index.js",
+    "build": "npm run schema-to-tsd && npm run compile-ts && npm run copy-schemas",
     "schema-to-tsd": "node scripts/build_schema_types",
     "compile-ts": "tsc --project .",
+    "copy-schemas": "cpy 'src/schemas/*.json' dist/schemas",
     "prepack": "npm run build",
-    "prepare": "npm run build",
     "lint": "tslint --project . && eslint test/*.test.js test/mocks/ test/helpers/",
     "test": "nyc mocha",
```

Since `npm test` now tests `dist/` not `src/`, what do you think of adding:

```suggestion
    "test": "npm build && nyc mocha",
```

to make sure the `npm test` doesn't get out of sync?